backend/src/main/resources/application.yml

```diff
@@ -1,5 +1,7 @@
 # Spring Boot configuration
 spring:
+  main:
+    allow-bean-definition-overriding: true
```

Comment:
Nemate nekdo zkusenost, cim by mohlo byt zpusobene bean definition overriding? Lze to povolit a vsechno slape, jen tuhle konfiguraci neznam.

Reply:
99% je to tim, ze `taskScheduler` je nazev, ktery se uz pouziva v ramci Spring Scheduler AutoConfiguration, tzn, ze kdybychom prejmenovali tridu z `TaskScheduler` na napr `LoonoTaskScheduler` to by mohlo fungovat